fhireval/test_suite/capability_statement/__init__.py

```diff
@@ -1,6 +1,6 @@
 
 # The bullet points associated with this set of tests in the evaluation document
-test_set_id = "2.1"
+test_set_id = "2.01"
```

Comment:
Well, the above may be correct. If so, don't mind the above.

Reply:
So, I had to reorganize the tests so that those that didn't rely on the full dataset to be loaded could be done within the scope of an empty one. So, some tests did experience a change in their id number (inside the document as well as inside the test suite). The zeros were added because those ids are sorted as strings and 10 and 11 were being run before 2, 3, etc.